sambacc/commands/main.py

```diff
@@ -98,10 +99,23 @@ def global_args(parser: Parser) -> None:
         "--samba-command-prefix",
         help="Wrap samba commands within a supplied command prefix",
     )
+    parser.add_argument(
+        "--skip-if",
+        dest="skip_conditions",
+        action="append",
+        type=skips.parse,
+        help=(
+            "Skip execution based on a condition. Conditions include"
+            " 'file:<path>', 'env:<var>(==|!=)<value>', and 'always:'."
```

Comment:
I know that we have `?` for more detailed explanation but shouldn't we mention about `file:!<path>`  also here?

Reply:
I'll add it as " 'file:[!]<path>', ... in a follow up PR. Sound OK?